holmes/plugins/toolsets/__init__.py

```
@@ -36,11 +36,12 @@
 from holmes.plugins.toolsets.opensearch.opensearch_logs import OpenSearchLogsToolset
 from holmes.plugins.toolsets.opensearch.opensearch_traces import OpenSearchTracesToolset
 from holmes.plugins.toolsets.prometheus.prometheus import PrometheusToolset
-from holmes.plugins.toolsets.rabbitmq.toolset_rabbitmq import RabbitMQToolset
-from holmes.plugins.toolsets.robusta.robusta import RobustaToolset
-from holmes.plugins.toolsets.atlas_mongodb.mongodb_atlas import MongoDBAtlasToolset
-from holmes.plugins.toolsets.runbook.runbook_fetcher import RunbookToolset
-from holmes.plugins.toolsets.servicenow.servicenow import ServiceNowToolset
+from holmes.plugins.toolsets.opensearch.opensearch import OpenSearchToolset
```

Ruff (0.12.2)

Redefinition of unused `OpenSearchToolset` from line 35

Remove definition: `OpenSearchToolset`

(F811)